src/components/App/AppIFrame.js

```diff
@@ -200,33 +158,23 @@ class AppIFrame extends React.Component {
     delete props.src
 
     return (
-      <React.Fragment>
-        {progressBar}
-        <StyledIFrame
-          name="AppIFrame"
-          allow="camera *; microphone *"
-          frameBorder="0"
-          onLoad={this.handleOnLoad}
-          ref={this.handleIFrameRef}
-          sandbox={SANDBOX}
-          style={{ display: show ? 'block' : 'none' }}
-          {...props}
-        />
-      </React.Fragment>
+      <iframe
+        name="AppIFrame"
+        allow="camera *; microphone *"
+        frameBorder="0"
+        onLoad={this.handleOnLoad}
+        ref={this.handleIFrameRef}
+        sandbox={SANDBOX}
+        style={{ display: show ? 'block' : 'none' }}
```

Comment:
Feels odd to have both `style` and `css` props, may I suggest using only one?

Reply:
Yes :+1: